File: app/src/main/java/com/OverSadBoy/samurairise/model/ModelContract.java
package com.OverSadBoy.samurairise.model;

import com.OverSadBoy.samurairise.model.database.Item;

import java.util.List;

public interface ModelContract {

    void addAlarm(Item item);
    void deleteAlarm(int id);
    List<Item> getAlarms();
}

File: app/src/main/java/com/OverSadBoy/samurairise/model/ModelContract.kt
package com.OverSadBoy.samurairise.model

import androidx.test.runner.AndroidJUnit4
import com.OverSadBoy.samurairise.model.database.Item
import org.junit.runner.RunWith
import java.util.*

interface ModelContract {
    open fun addAlarm(item: Item?)
    open fun deleteAlarm(id: Int)
    open fun getAlarms(): ArrayList<Item?>?
}